Runtime/Interaction/StaticMesh/StaticMeshCollidersManager.cs

```diff
@@ -0,0 +1,183 @@
+using System;
+using System.Collections.Generic;
+using Unity.Collections;
+using Unity.Jobs;
+using UnityEngine;
+using UnityEngine.Jobs;
+
+namespace ubco.ovilab.HPUI.StaticMesh
+{
+    /// <summary>
+    /// Component to manage the colliders for a given rectangular custom mesh, based on <see cref="Interaction.DeformableSurfaceCollidersManager"/>
+    /// </summary>
+    public class StaticMeshCollidersManager : MonoBehaviour
+    {
+        [SerializeField] private VertexRemapData vertexRemapData;
```

Comment:
I still see no reason for this to be a scriptable object. This is an internal representation that does not need to be exposed to the user at all.

Reply:
I will fix this later promise 🤞

Reply:
fixed